test/integration/targets/collections_runtime_pythonpath/runme.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -mvEeuxo pipefail
+
+
+export PIP_DISABLE_PIP_VERSION_CHECK=1
+
+
+function _cleanup() {
+    echo Cleanup: uninstalling ansible-collections.python.dist...
+    python -m pip.__main__ uninstall -y ansible-collections.python.dist
+}
+trap _cleanup INT TERM EXIT QUIT
```

Comment:
Remove this cleanup so that the contents of `OUTPUT_DIR` can be inspected if needed after the test ends. The cleanup is redundant, since ansible-test ensures the directory is clean before running each test. The contents are intentionally left behind after each test for easier troubleshooting.